package.json

```diff
@@ -86,12 +96,13 @@
     "vscode": "^1.0.3"
   },
   "dependencies": {
-    "import-sort": "3.2.0",
-    "import-sort-config": "^3.2.0",
-    "import-sort-parser-babylon": "^3.2.0",
-    "import-sort-parser-typescript": "^3.2.0",
-    "import-sort-style-eslint": "^3.0.0",
-    "import-sort-style-module": "^3.0.0",
-    "import-sort-style-module-compact": "^1.0.0"
+    "import-sort": "4.2.0",
+    "import-sort-config": "^4.2.0",
+    "import-sort-parser-babylon": "^4.2.0",
+    "import-sort-parser-typescript": "^4.2.0",
+    "import-sort-style-eslint": "^4.0.0",
+    "import-sort-style-module": "^4.0.0",
+    "import-sort-style-module-compact": "^1.0.0",
+    "import-sort-style-module-scoped": "^1.0.1"
```

Comment:
This is a nice addition, I didn't know this style. 👍

Reply:
I created it - module did not support scoped repos (e.g. `@something/blah`).

Reply:
Nice idea! We can also fix that directly in [`module-compact`](https://github.com/amatiasq/import-sort-style-module-compact) but let's add this one anyway.